permits/management/commands/fixturize.py

```diff
@@ -602,3 +603,37 @@ def create_template_customization(self):
             application_subtitle="Demandes en lignes",
             application_description="Demandes concernant l' <i>administration</i>",
         )
+
+    def create_document_types(self, wot):
+        document_types = [
+            (
+                "{} Parent #1".format(wot.pk),
```

Comment:
You could use the `range()` Python built-in function to achieve the document type creation, instead of manually typing the Children 😛